README.md

```diff
@@ -216,6 +216,10 @@ __Sample of a real Grabbit Job status__
 
 ![alt text](assets/ExampleGrabbitStatus.jpg)
 
+Two loggers are predefined for Grabbit. One for Grabbit Server and the other for Grabbit Client. 
```

Comment:
Users would care a great deal about the log files...

The `README.md` file is being rendered from the context of https://github.com/TWCable/grabbit, so you can use a relative path to `blob/master/grabbit/src/main/content/SLING-INF/content/apps/grabbit/config/org.apache.sling.commons.log.LogManager.factory.config-com.twcable.grabbit.client.batch.xml`